API/resources/resources.json

```diff
@@ -37,10 +37,15 @@
       },
       "config": [
         {
-          "condition": "'%{app}' == 'iotjs'",
+          "condition": "'%{app}' == 'iotjs' and '%{build-type}' == 'release'",
           "src": "%{config}/tizenrt-iotjs.config",
```

Comment:
Please rename the `tizenrt-iotjs.config` file to `tizenrt-iotjs-release.config`.